### ISSUE TITLE
Allow option to use a specific R version

### ISSUE THREAD

**Issue by rutujact** [CredibilityLab/groundhog#68]
Hi,

I just finished working on a long and complex project. Just around then I heard about groundhog. 
For my future self (and other potential project users) I want to switch over to all instances of install.packages(pkg) and library(pkg) to groundhog.library(pkg, groundhog.day).

The packages I have used must have been installed on different occasions over the past year. And I used R version 3.6.3.
Until yesterday everything worked fine (without groundhog). 

Groundhog let's me set the groundhog.day to a date when R v3.6 was current (e.g. groundhog.day = "2020-04-01") But then some of my recently downloaded package functions do not work. For example, I get the following message when packages are entirely downloaded with groundhog with groundhog.day = "2020-04-01": 

`a.b.r2 <- signif(broom::glance(k_by_psi.models$A.B)$adj.r.squared, 2)`
> Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
>   namespace ‘rlang’ 0.4.5 is already loaded, but >= 0.4.9 is required

If I use groundhog.day >=  "2020-05-01", groundhog forces me to a later R version. See below message:

`groundhog.library('devtools', "2020-09-01)`

> groundhog says:
> You are using R-3.6.3 and the current R version for the data you entered:'2020-09-01' was R-4.0.
> To ensure reproducibility you must use groundhog.library() with a version of R that is as least
> as recent as the entered date (i.e., R>=4.0)
>    ----------------- Package 'devtools' NOT LOADED ----------------

Is there a workaround to force groundhog to use the specific R version I have been using (that is, Rv3.6.3)? I am hoping that with that the rlang error will go away. (I would, of course, leave a note in the code for my future self pointing to the specific R version the code was tested on.) Or what may be a better solution if I want to use groundhog nevertheless?

**Comment by urisohn** [comment]
Currently not possible to do this (bypass R version check).
Very easy to implement but we put it there to prevent unexpected incompatibilities.
Open to possibility though. Why wouldn't you update to the current version of R? It is easy on a mac have both readily available
https://groundhogr.com/many

**Comment by rutujact** [comment]
I was hesitant to update R version, for I wasn't sure if a same R package version works differently with different R versions or some R packages may declare that they are not available for the new R version. But I updated to R 4.0.3 anyway. I had to update how some of the tidyverse command worked (most likely because I had not recorded which of its version I was using prior to using groundhog). But everything now works with groundhog and it will be reproducible hereafter, so I am glad.
